lib/package.gi

```diff
@@ -1099,6 +1099,19 @@ InstallGlobalFunction( IsPackageMarkedForLoading, function( name, version )
 ##
 #F  DefaultPackageBannerString( <inforec> )
 ##
+
+DeclareUserPreference( rec(
+  name:= "ShortBanners",
+  description:= [
+    "If this option is set to <K>true</K>, only the first line of each \
+package banner (including the name, version and description of the package) \
+is displayed."
```

Comment:
That is only true for packages that use the default banner, is it? Or do you also do something for packages with a custom banner? But then this text also ought to be adjusted.

Reply:
Ah, I wasn't aware of custom package banners. Fixed.